torch/_dynamo/variables/functions.py

```diff
@@ -460,17 +476,38 @@ def export_freevars(self, parent, child):
                 parent.symbolic_locals[var] = child.symbolic_locals[var]
 
     def reconstruct(self, codegen):
-        flags = 0x00
+        create_nested_fn_name = unique_id("__create_nested_fn")
+        codegen.tx.output.install_global(
+            create_nested_fn_name, disable(_create_nested_fn)
+        )
+
+        codegen.extend_output(
+            [
+                codegen.create_load_global(
+                    create_nested_fn_name, push_null=False, add=True
+                ),
+            ]
+        )
```

Comment:
I think this can just be:
```suggestion
        cg.load_import_from(__name__, "_create_nested_fn")
```

Reply:
Yes, it works! And more concise!